client/src/cmdlfnexwatch.c

```diff
@@ -368,6 +407,17 @@ static int CmdNexWatchClone(const char *Cmd) {
         raw[10] |= nexwatch_checksum(magic, cn, parity);
     }
 
+    if (use_psk2) {
+        blocks[0] = 270464;
+        uint8_t * res_shifted = malloc(96 * sizeof(uint8_t));
```

Comment:
1. we use calloc in the repo instead of malloc.
2. I see no free of this allocated memory

Reply:
Ok, we fix it this afternoon then !